models/index.js

```diff
@@ -16,6 +16,26 @@ var env       = process.env.NODE_ENV || 'development';
 var config    = require(__dirname + '/../config/config.js')[env];
 var db        = {};
 
+function UserException(message) {
```

Comment:
Does this work instead of creating our own function? I don't know whether or not it does, but I'd prefer not to create our own function https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw